pages/concepts/protocols/portal-sub-protocols.md

```diff
@@ -0,0 +1,12 @@
+## Portal Network Sub-protocols
+
+The Portal Network is really a collection of separate networks that each handle some specific subset of the total Ethereum data.
+Portal clients have to conform to specific sub-protocols to enable them to participate in each of these networks.
+
+The Portal sub-protocols are:
+
+- [History](./portal-sub-protocols/history.md)
+- [Beacon](./portal-sub-protocols/beacon.md)
+- [State](./portal-sub-protocols/state.md)
+- [Transaction gossip](./portal-sub-protocols/transaction-gossip.md)
+- [Canonical transaction index](./portal-sub-protocols/canonical-tx-index.md)
```

Comment:
I think that we should leave these out for now.  Doing them is probably contingent on them being useful for known/desired use cases by those actually adopting our protocols and I'd like to avoid committing ourselves towards them at this stage.

Reply:
ok fair enough - will remove tx gossip and canonical tx index pages

Reply:
done